ttnn/cpp/ttnn/operations/data_movement/copy/copy_pybind.cpp

```diff
@@ -140,23 +140,27 @@ void py_bind_assign(py::module& module) {
                 const ttnn::Tensor& input,
                 const ttnn::MemoryConfig memory_config,
                 const std::optional<const ttnn::DataType> dtype,
+                std::optional<ttnn::Tensor> &optional_output_tensor,
                 uint8_t queue_id) {
-                    return self(queue_id, input, memory_config, dtype);
+                    return self(queue_id, input, memory_config, dtype, optional_output_tensor);
                 },
                 py::arg("input_tensor").noconvert(),
                 py::kw_only(),
                 py::arg("memory_config"),
                 py::arg("dtype") = std::nullopt,
+                py::arg("optional_tensor") = std::nullopt,
```

Comment:
kwarg name "output_tensor" is used commonly in TTNN

Reply:
Done